### ISSUE TITLE
Remove 'back to content' on accessibility page

### ISSUE THREAD

**Issue by GCHQ-Developer-465** [mi6/ic-design-system#732]
## Summary

Remove 'back to content' on accessibility page. 

## 💬 Description

When mentioning the reference there is a 'back to content' link on the [accessibility page](https://design.sis.gov.uk/accessibility/introduction/#ref-2). This is repeated and is unnecessary when the content is so short and is just above the references. This should be removed. 

## 💰 Use value

This is duplicating and is unnecessary, therefore should be removed. 

## Additional info

Please see screenshot below:
<img width="867" alt="Screenshot 2023-12-18 at 13 06 55" src="https://github.com/mi6/ic-design-system/assets/111882114/6de04e12-15bc-49d3-a2c8-8c370d5f6c28">
